downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,62 @@
+[id="eda-custom-credential-types"]
+
+= Custom credential types
+
+As a system administrator, you can define a custom credential type works in ways similar to existing credential types in a standard format using a YAML or JSON-like definition. For example, a custom credential type can inject an API token for a third-party web service into an environment variable for your rulebook or custom inventory script to consume.
+
+Each credential type displays its own unique configurations in the Input Configuration field and the Injector Configuration field, if applicable. Custom credentials support the following ways of injecting their authentication information:
+
+* Environment variables
+* Ansible extra variables
+* File-based templating, which means generating .ini or .conf files that contain credential values
+
+//[J.Self] Reviewers: please confirm if this is true. I borrowed it from automation controller content, per suggestions from several SMEs and one of our principal tech writers. I just want to confirm technical accuracy here, especially the last line as we might not have "machine credentials".
+You can attach one SSH and multiple cloud credentials to a job template. Each cloud credential must be of a different type. Only one of each type of credential is permitted. Vault credentials and machine credentials are separate entities.
```

Comment:
@jself-sudoku You can attach 1 or more cloud credentials, vault credentials and RH AAP Credentials. SSH I think pertains to AAP and doesn't apply to EDA. In EDA SSH Keys are only used inside Source Credential for Project Syncs.

Reply:
Noted.